<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Regression" duration-ms="81" started-at="2020-04-12T14:19:31Z" finished-at="2020-04-12T14:19:31Z">
    <groups>
      <group name="sanity">
        <method signature="LoginTest.verifyUserShouldNavigateToLoginPage()[pri:0, instance:com.demo.nopcommerce.testsuite.LoginTest@12843fce]" name="verifyUserShouldNavigateToLoginPage" class="com.demo.nopcommerce.testsuite.LoginTest"/>
      </group> <!-- sanity -->
    </groups>
    <test name="Login Test" duration-ms="81" started-at="2020-04-12T14:19:31Z" finished-at="2020-04-12T14:19:31Z">
      <class name="com.demo.nopcommerce.testsuite.LoginTest">
        <test-method status="FAIL" signature="verifyUserShouldNavigateToLoginPage()[pri:0, instance:com.demo.nopcommerce.testsuite.LoginTest@12843fce]" name="verifyUserShouldNavigateToLoginPage" duration-ms="12" started-at="2020-04-12T14:19:31Z" finished-at="2020-04-12T14:19:31Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.demo.nopcommerce.utility.Utility.clickOnElement(Utility.java:57)
at com.demo.nopcommerce.pages.HomePage.clickOnLoginLink(HomePage.java:16)
at com.demo.nopcommerce.testsuite.LoginTest.verifyUserShouldNavigateToLoginPage(LoginTest.java:19)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyUserShouldNavigateToLoginPage -->
      </class> <!-- com.demo.nopcommerce.testsuite.LoginTest -->
    </test> <!-- Login Test -->
  </suite> <!-- Regression -->
</testng-results>
